infra/wca_on_rails/production/auxiliary_services.tf

```diff
@@ -0,0 +1,131 @@
+  # as in https://aws.amazon.com/blogs/containers/new-using-amazon-ecs-exec-access-your-containers-fargate-ec2
+  execution_role_arn = aws_iam_role.task_execution_role.arn
+  task_role_arn      = aws_iam_role.task_role.arn
+
+  cpu = "1024"
+  memory = "1548"
+
+  container_definitions = jsonencode([
+    {
+      name              = "sidekiq-main"
+      image             = "${var.shared.ecr_repository.repository_url}:sidekiq-production"
+      cpu    = 512
+      memory = 1024
+      portMappings = []
+      logConfiguration = {
+        logDriver = "awslogs"
+        options = {
+          awslogs-group         = aws_cloudwatch_log_group.this.name
+          awslogs-region        = var.region
+          awslogs-stream-prefix = var.name_prefix
+        }
+      }
+      environment = local.rails_environment
+      healthCheck       = {
+        command            = ["CMD-SHELL", "pgrep ruby || exit 1"]
+        interval           = 30
+        retries            = 3
+        startPeriod        = 60
+        timeout            = 5
+      }
+    },
+    {
+      name              = "pma-production"
+      image             = "${var.shared.ecr_repository.repository_url}:pma"
+      cpu    = 256
+      memory = 512
+      portMappings = [{
+        # The hostPort is automatically set for awsvpc network mode,
+        # see https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_PortMapping.html#ECS-Type-PortMapping-hostPort
+        containerPort = 80
+        protocol      = "tcp"
+      }]
+      logConfiguration = {
+        logDriver = "awslogs"
+        options = {
+          awslogs-group         = aws_cloudwatch_log_group.this.name
+          awslogs-region        = var.region
+          awslogs-stream-prefix = var.name_prefix
+        }
+      }
+      environment = local.pma_environment
+      healthCheck       = {
+        command            = ["CMD-SHELL", "curl -f http://localhost/LICENSE || exit 1"]
+        interval           = 30
+        retries            = 3
+        startPeriod        = 60
+        timeout            = 5
+      }
+    }
+    ])
+
+  tags = {
+    Name = var.name_prefix
+  }
+}
+
+data "aws_ecs_task_definition" "auxiliary" {
+  task_definition = aws_ecs_task_definition.auxiliary.family
+}
+
+resource "aws_ecs_service" "auxiliary" {
+  name                               = "${var.name_prefix}-auxiliary-services"
+  cluster                            = var.shared.ecs_cluster.id
+  # During deployment a new task revision is created with modified
+  # container image, so we want use data.aws_ecs_task_definition to
+  # always point to the active task definition
+  task_definition                    = data.aws_ecs_task_definition.auxiliary.arn
+  desired_count                      = 0
+  scheduling_strategy                = "REPLICA"
+  deployment_maximum_percent         = 200
+  deployment_minimum_healthy_percent = 50
+  health_check_grace_period_seconds  = 0
```

Comment:
(see staging)

Reply:
doesn't need it because sidekiq and pma boot fast